Standards/scs-0001-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,305 @@
+| `type` | REQUIRED | one of `Procedural`, `Standard`, or `Decision Record` |
+| `status` | REQUIRED | one of `Draft`, `Stable`, `Deprecated`, or `Rejected` |
+| `track` | REQUIRED | one of `Global`, `IaaS`, `Ops`, `KaaS` |
+| `obsoleted_at` | REQUIRED if `status` is `Deprecated` | ISO formatted date indicating the date after which the deprecation is in effect |
+| `stabilized_at` | REQUIRED if `status` was ever `Stable` | ISO formatted date indicating the date after which the document was considered stable |
+| `rejected_at` | REQUIRED if `status` is `Rejected` | ISO formatted date indicating the date on which the document was rejected |
+| `replaced_by` | RECOMMENDED if `status` is `Deprecated` or `Rejected`, FORBIDDEN otherwise | List of documents which replace this document. |
+| `replaces` | OPTIONAL | List of documents which this document replaces. |
+
+## Sections
+
+Each document SHOULD have the following sections:
+
+- An *Introduction* providing context on the document and linking to other relevant materials.
+- A *Motivation* section which details why this document or the thing it describes is necessary.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A *Design Considerations* section for Standard type documents,
+  which details other choices
+  which have been considered for the specific feature
+  but were ultimately rejected.
+- An *Open Questions* section which links to issues
+  detailing any open discussion points with respect to a document.
+  This section is RECOMMENDED during the discussion phase (pre 1.0.0)
+  as a "table of contents" of things to work on in that context.
+
+# Process
+
+The lifecycle of an SCS document goes through the following phases:
+Proposal, Draft, Stable, Deprecated, and Rejected.
+
+All decisions for phase transitions follow loose consensus,
+where the group which has to form the consensus depends on the `track` of the document:
+
+- IaaS: The team working on infrastructure-as-a-service topics
+- KaaS: The team working on Kubernetes-as-a-service topics
+- Ops: The team working on operations topics
+- IAM: The team working on idenity and access management topics
+- Global: The entire SCS community.
+
+## Proposal phase
+
+### Proposal of a new document
+
+To propose a new SCS document,
+a community participant creates a pull request on GitHub
+against the [Docs repository in the SovereignCloudStack organisation](https://github.com/SovereignCloudStack/Docs).
+
+The pull request MUST add exactly one SCS document,
+in the `Standards` folder.
+In the proposal phase,
+the document number MUST be replaced with `xxxx` in the file name.
+The major version MUST be 1.
+
+For a document with a slugified title `flavor-naming`,
+the file name would for instance be `scs-xxxx-v1-flavor-naming.md`.
+
+The metadata MUST indicate the intended `track` and `type` of the document,
+and the `status` MUST be set to `Draft`
+and the major number in the `version` SHOULD be `0`.
```

Comment:
I'm a bit unsure whether there is a difference between the major version in the file name which MUST be 1 on the one hand, and the major number in `version` which SHOULD be `0`. Are they major versions/numbers with distinct meaning?

Furthermore, is `version` a metadata field in the markdown headers? If yes, it is missing in the above table of metadata fields.

Reply:
Good catch. This is a leftover, there should be no version field at all anymore (I had a previous iteration with a separate version field and that's gone now, I just missed that bit of text).